### ISSUE TITLE
Non interactive breaks with v2.7.0

### ISSUE THREAD

**Issue by swernerx** [webpro/release-it#87]
Hi!

I'll use the tool in most of my open source projects to release both GitHub and NPM releases which was super useful. Unfortunately with the current version it somehow breaks. With debugging on and in dry mode it's easy to see the error:

```
sebastian in ~/Workspace/sebastian-software/prepublish on master*
⚡ release-it --github.release --npm.publish --non-interactive -d -e
{ _: [],
  github: 
   { release: true,
     releaseName: 'Release %s',
     preRelease: false,
     tokenRef: 'GITHUB_TOKEN' },
  g: true,
  npm: 
   { publish: true,
     version: '0.15.2',
     name: 'prepublish',
     private: false,
     publishPath: '.',
     tag: 'latest',
     forcePublishSourceRepo: false },
  p: true,
  'non-interactive': true,
  n: true,
  d: true,
  'dry-run': true,
  e: true,
  debug: true,
  i: undefined,
  increment: 'patch',
  src: 
   { commitMessage: 'Release %s',
     tagName: '%s',
     tagAnnotation: 'Release %s',
     pushRepo: null,
     beforeStartCommand: false,
     beforeStageCommand: false,
     afterReleaseCommand: false },
  dist: 
   { commitMessage: 'Release %s',
     repo: false,
     stageDir: '.stage',
     baseDir: 'dist',
     files: [ '**/*' ],
     pkgFiles: null,
     tagName: '%s',
     tagAnnotation: 'Release %s',
     beforeStageCommand: false,
     afterReleaseCommand: false },
  pkgFiles: [ 'package.json' ],
  verbose: true,
  force: false,
  prereleaseId: null,
  buildCommand: false,
  changelogCommand: 'git log --pretty=format:\'* %s (%h)\' [REV_RANGE]',
  requireCleanWorkingDir: false,
  prompt: 
   { src: 
      { status: false,
        commit: true,
        tag: true,
        push: true,
        release: true,
        publish: false },
     dist: 
      { status: false,
        commit: true,
        tag: true,
        push: true,
        release: true,
        publish: false } },
  name: 'prepublish' }
[dry-run] git config --get remote.origin.url
https://github.com/sebastian-software/prepublish.git
[dry-run] git show-ref --tags --quiet --verify -- "refs/tags/0.15.2"
[dry-run] git log --pretty=format:'* %s (%h)' 0.15.2...HEAD

Release source repo
[dry-run] git rev-parse --git-dir
.git
[dry-run] bump package.json 0.15.3
No build command was provided.
[dry-run] git add package.json
[dry-run] git diff-index --name-only HEAD --exit-code
[dry-run] git commit  --message="Release 0.15.3" .
[dry-run] git tag  --annotate --message="Release 0.15.3" 0.15.3
[dry-run] git push 
[dry-run] git push --follow-tags  
ERROR Path must be a string. Received undefined
TypeError: Path must be a string. Received undefined
    at assertPath (path.js:7:11)
    at Object.basename (path.js:1355:5)
    at parseRemoteUri (/usr/local/lib/node_modules/release-it/node_modules/parse-repo/index.js:40:24)
    at release (/usr/local/lib/node_modules/release-it/lib/git.js:201:16)
    at /usr/local/lib/node_modules/release-it/node_modules/when/sequence.js:33:22
    at tryCatchResolve (/usr/local/lib/node_modules/release-it/node_modules/when/lib/apply.js:46:23)
    at callAndResolve (/usr/local/lib/node_modules/release-it/node_modules/when/lib/apply.js:30:12)
    at callAndResolveNext (/usr/local/lib/node_modules/release-it/node_modules/when/lib/apply.js:40:4)
    at Fold.fulfilled (/usr/local/lib/node_modules/release-it/node_modules/when/lib/makePromise.js:767:11)
    at tryCatchReject (/usr/local/lib/node_modules/release-it/node_modules/when/lib/makePromise.js:845:30)
```

Do you have any idea where this is coming from?

Going back to 2.6.1 everything works fine.

Thanks for your work!

**Comment by webpro** [comment]
Yes, I've seen this myself as well after the release. Not sure what it is yet, but I'm going to look into it soon.

**Comment by webpro** [comment]
Fixed in [2.7.1](https://github.com/webpro/release-it/releases/tag/2.7.1)
